test/acceptance/test/add_tests.go

```diff
@@ -337,14 +340,20 @@ var _ = Describe("Weave GitOps Add Tests", func() {
 		})
 
 		By("And I run 'wego app add dry-run' command", func() {
-			session = runWegoAddCommandAndReturnSession(repoAbsolutePath, addCommand, WEGO_DEFAULT_NAMESPACE)
+			addCommandOutput, _ = runWegoAddCommandWithOutput(repoAbsolutePath, addCommand, WEGO_DEFAULT_NAMESPACE)
 		})
 
 		By("Then I should see dry-run output with specified: url, namespace, branch", func() {
-			Eventually(session).Should(gbytes.Say("using URL: '" + url + "'"))
-			Eventually(session).Should(gbytes.Say("Checking cluster status... WeGOInstalled"))
-			Eventually(session).Should(gbytes.Say(`apiVersion:.*\nkind: GitRepository\nmetadata:\n\s*name: ` + appName + `\n\s*namespace: ` + WEGO_DEFAULT_NAMESPACE + `[a-z0-9:\n\s*]+branch: ` + branchName + `\n\s*.*\n\s*name: ` + appName + `\n\s*url: ` + url))
-			Eventually(session).Should(gbytes.Say(
+			Eventually(addCommandOutput).Should(MatchRegexp(`using URL: '` + url + `'`))
```

Comment:
Validating specific command-line output will make these tests very brittle. I realize that much of this suite already does that, so its not necessarily unique to this PR (nor should it block merging).

Reply:
I realize that, but at the same time these outputs are user-facing scenarios. So, they need to be tested one way or another. These aren't part of the Smoke Tests, so they won't be blocking any merges. Any changes to these outputs may fail acceptance tests, but that just means we'll have to adjust the assertions being made.